src/core/merkle/MerkleTreeBuilder.ts

```diff
@@ -72,6 +71,14 @@ export class MerkleTreeBuilder {
                     `${ window.location.href.replace("/index.html", "") }/assets/scripts/SHA1PRNG.js`,
                     `${ window.location.href.replace("/index.html", "") }/assets/scripts/LamportGenerator.js`
                 ];
+            } else if (!window.location.protocol.includes("http")) {
+                // Browser extensions requires the scripts to be loaded as shown below
+                scripts = [
+                    `${ window.location.protocol }//${ window.location.host }/www/assets/scripts/sjcl.js`,
+                    `${ window.location.protocol }//${ window.location.host }/www/assets/scripts/seedrandom.min.js`,
```

Comment:
I don't believe the seedrandom library is needed anymore. Please double check and remove this line if this is true.